src/sentry/static/sentry/app/views/settings/project/projectTeams.jsx

```diff
@@ -0,0 +1,147 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import createReactClass from 'create-react-class';
+
+import {t} from '../../../locale';
+import ApiMixin from '../../../mixins/apiMixin';
+import AsyncView from '../../asyncView';
+import Confirm from '../../../components/confirm';
+import IndicatorStore from '../../../stores/indicatorStore';
+import SettingsPageHeader from '../components/settingsPageHeader';
+
+const TeamRow = createReactClass({
+  displayName: 'TeamRow',
+
+  propTypes: {
+    orgId: PropTypes.string.isRequired,
+    projectId: PropTypes.string.isRequired,
+    team: PropTypes.object.isRequired,
+    access: PropTypes.object.isRequired,
+    onRemove: PropTypes.func.isRequired,
+  },
+
+  mixins: [ApiMixin],
+
+  getInitialState() {
+    return {
+      loading: false,
+      error: false,
+    };
+  },
+
+  handleRemove() {
+    if (this.state.loading) return;
+
+    let loadingIndicator = IndicatorStore.add(t('Saving changes..'));
+    let {orgId, projectId, team} = this.props;
+    this.api.request(`/projects/${orgId}/${projectId}/teams/${team.slug}/`, {
+      method: 'DELETE',
+      success: (d, _, jqXHR) => {
+        this.props.onRemove();
+        IndicatorStore.remove(loadingIndicator);
+      },
+      error: () => {
+        this.setState({
+          error: true,
+          loading: false,
+        });
+        IndicatorStore.remove(loadingIndicator);
+      },
+    });
+  },
+
+  render() {
+    let team = this.props.team;
+    return (
+      <tr>
```

Comment:
Similar to in parent view, let's use `<Row>` here like in https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/views/settings/account/apiApplications.jsx#L76